symphony/versions.json

```diff
@@ -79,6 +73,30 @@
       },
       "previous_version_name": "v28",
       "next_version_name": ""
+    },
+    {
+      "name": "v1.0.7",
+      "tag": "v1.0.7",
+      "height": 3333145,
```

Comment:
This don't look to be the right block height. It's already used for previous versions.
(we'll be adding validation that the same block height isn't used on multiple version objects)